xmodule/tests/test_library_tools.py

```diff
@@ -94,3 +99,105 @@ def test_import_from_blockstore(self):
         imported_html_block = self.store.get_item(imported_unit_block.children[0])
         assert 'Hello world' not in imported_html_block.data
         assert 'Foo bar' in imported_html_block.data
+
+    def test_get_v2_library_version(self):
+        """
+        Test get_library_version for V2 libraries.
+
+        Covers getting results for either library key as a string or LibraryLocatorV2.
+
+        NOTE:
+            We don't publish library updates so the library version will always be 0.
```

Comment:
> We don't publish library updates so the library version will always be 0.

Is this going to change, or are libraries going to remain unversioned?

Reply:
Libraries versioning works, but this isn't checked because the test goal here is to prove that the version could be received either for the id as a string or as a LibraryLocatorV2. 
Publishing a library is a part of the content libraries functionality, not the library-content/library-sourced xblock so I don't test it here.

Reply:
Ah ok, this language is not wholly correct then. I knew libraries were versioned, I didn't know about the xblock level. Maybe `We are not testing the library version here, so it will be 0` is more informative and less confusing.

Reply:
Ok
@connorhaugh don't you mind if I ask @UvgenGen to update this in his future PRs? I propose that because his work is based on this branch and I don't want him for rebasing several feature branches because of comment change )

Reply:
Yes that is ok!